File: j/java/src/main/java/sh/foxboy/command/PingCommand.java
package sh.foxboy.command;

import net.dv8tion.jda.api.entities.Message;
import sh.foxboy.api.command.JDACommand;

public class PingCommand extends JDACommand {

  public PingCommand(String name) {
    super(name);
  }

  @Override
  public void execute(Message message, String commandLabel, String[] args) {
    message.getChannel().sendMessage(message.getAuthor().getAsMention() + " Pong! (Command: " + commandLabel
        + ", Args: " + String.join(" ", args) + ")").queue();
  }

}

File: k/kotlin/src/main/kotlin/sh/foxboy/commands/PingCommand.kt
package sh.foxboy.commands

import net.dv8tion.jda.api.entities.Message
import sh.foxboy.api.command.Command

class PingCommand(name: String) : Command(name) {
    override fun execute(message: Message, commandLabel: String, args: Array<String?>?) {
        message.reply("Pong! (Command: ${commandLabel}, Args: ${args?.joinToString()})").queue()
    }
}